src/alwaysAck.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+/* @flow */
+
+import type { MessageHandler } from './index'
+
+import { ackOnComplete, ackOnError } from './ackOn'
+import parseAndHandleMessage from './parseAndHandleMessage'
+
+export default (handleError:(e:mixed) => Promise<mixed>, f:(message:Object) => Promise<mixed>): MessageHandler =>
+  parseAndHandleMessage(x => x, ackOnError(handleError), ackOnComplete, f)
```

Comment:
I agree--not arguing against that.  However, it seemed like the premise was that `parseAndHandleMessage` could be implemented in terms of `alwaysAck` ("the other way around").  As the code stands right now, I don't think it can.  Do you have a suggestion for how to get to a better spot?